README.md

```diff
@@ -1,38 +1,50 @@
-jupyter-vim-binding
-========================
-Enable Vim key bindings in [jupyter](https://jupyter.org/) or [IPython notebook](http://ipython.org/notebook.html).
+#Jupyter VIM Binding
+
+This extension enables VIM keybindings in [Jupyter Notebook](https://jupyter.org/) (4.0+).
```

Comment:
Please add `or [IPython notebook](http://ipython.org/notebook.html).` while IPython notebook -> Jupyter migration has not been fully finished.